messages/package_version_create.md

```diff
@@ -194,3 +194,15 @@ The version number is required and was not found in the options or in package js
 # missingConnection
 
 A connection is required.
+
+# IdUnavailableWhenQueued
+
+Request is queued. ID unavailable.
+
+# IdUnavailableWhenInProgress
+
+Request is in progress. ID unavailable.
+
+# IdUnavailableWhenError
+
+ID Unavailable
```

Comment:
These should be reviewed by Penelope (packaging doc writer).  Ideally, messages are clear and errors provide suggestions on what they can do to fix things or how to proceed.  E.g., what ID is this referring to?